modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize and configure an existing project access roles name by editing the YAML code of `console.openshift.io/display-name` of your cluster.
+
+.Procedure
+To customize and configure the cluster roles of your project:
```

Comment:
Did you think it's clear here that all projects are affected if the user changes the cluster configuration?

```suggestion
To customize and configure the cluster roles of all projects:
```

Reply:
In my understanding, it was only for the current project. And thanks for clarifying that it is for all the projects. I'll update the sentence.